### ISSUE TITLE
Feature request: Unifi Backup

### ISSUE THREAD

**Issue by Ilovegym66** [simatec/ioBroker.backitup#1024]
from here: 
https://github.com/ioBroker/AdapterRequests/issues/797

**Comment by mcm1957** [comment]
And hier the full text from issue https://github.com/ioBroker/AdapterRequests/issues/797


**What kind of device or service would you like to see an adapter for?**

I would like to have an Adapter which can back up the database of unifi devices.
There is the Database from the unifi Controller, which is stored locally on the device. If the device is broken you can’t replace the hardware without the backup. There is NO function to save the file to any external device.
There are services online which can backup the database but you need to „open“ the firewall to get it working and i tried two services without success.

https://www.ui.com/

**Is the device connected to the internet or only available on a local network?**

LAN

**Is an official App or Website available?**
https://www.ui.com/

**Is an official API including documentation available?**
If yes, please add links and information[...]
Probably, as there is the unifi adapter which connects to thr mongodb database.

**Are other libraries for an integration available?**
Not sure.

**Is this device already integrated in other Smart Home systems?**
Yes, into iobroker with the unifi adapter

**Is this device already integrated in homebridge? No.

**Comment by simatec** [comment]
I'm not sure whether this backup is in the right place in Backitup.
Backitup deals with all backup options that are related to the iobroker.
Theoretically, the adapter is already extremely advanced.

In my opinion, a Unify backup does not fit into the concept of backitup.
We would be happy to discuss this topic in the dev meeting.

**Comment by simatec** [comment]
How should I understand the answer? ... I'll open it again, as we're taking this into the dev meeting as a discussion

**Comment by DutchmanNL** [comment]
> shaking my head and close it

Please dont get me wrong, this is an internal developer discussion how to continue not mentioned to close your request in general

@simatec yes fully agree, let's discuss in dev meeting and provide feedback back to this issue

For all readers; dev meeting is scheduled for 29th of November, we will park this issue and update you first week of December about our next steps. We need some "nurds heads together" discussion about this :) 

Cheers, Dutchman

**Comment by Ilovegym66** [comment]
Hi, sorry, I know, Xmas is in some weeks... :-) provide a political strategy for the Adapter.. I know, it starts as a easy backup automatic solution, and now it's a all-around-the machine-backup solution.. so it's difficult to say, what can / must/ need / should be added, if it make sense.. :-)

Ask the users in the forum / discord / facebook / telegram, what is needed to backup additional to the iobroker.

**Comment by DutchmanNL** [comment]
> Hi, sorry, I know, Xmas is in some weeks... :-) provide a political strategy for the Adapter.. I know, it starts as a easy backup automatic solution, and now it's a all-around-the machine-backup solution.. so it's difficult to say, what can / must/ need / should be added, if it make sense.. :-)
> 
> 
> 
> Ask the users in the forum / discord / facebook / telegram, what is needed to backup additional to the iobroker.
> 
> 

Fully agree, scope of this discussion is not to do it or not, more if we should "blow up" this adapter with all kinds of of non-ioBroker related aspects or start a separate development/instanze

This has nothing to do with generic functionality more how to manage and maintain the code 😉

**Comment by DutchmanNL** [comment]
> 2\. How much effort, especially on the UI part, is necessary? Since Unify devices are Access points (right?) you clearly have a lot of UI components already for a single device, and many will have multiple devices → it probably needs a new page in the configuration just for unify...

no that's the point, unify can be set-up in 3 flavours

- AP's only be controlled by an AP (no backup possible)
- central management by Unify Controller (original scope of the issue)
- central management by an dream machine

but please, this discussion is now moving on git & our forum and we already agreed to have it detailed discussed in our Development meeting.
I would like to suggest to centralise this (and avoid incorrect interpretations of the technology or how it can be used) and have it calcified next week (29th of November)

To summarise again, during our meeting we will discuss

- what it he scope of backlit up
- we can use unified as an example
- if we want to continue with unified, as mention in my previous comment, there are 2 different scenarios (Unify control, central approach based on native installation or docker, Unify Dream machine which is an OutOfThe Box manage all device)
-
